### PR TITLE
Rename and release

### PR DESCRIPTION
**Pull request by autra**
This changes the link after the repo rename + bumps the version

### CODE REVIEW

**Comment by autra** [comment]
@iTowns/reviewers please r?

File: BUILDING.md [hunk 1]
```diff
@@ -19,23 +19,11 @@ iTowns is managed on the GitHub platform. You should start with [creating a GitH
...
-   * Yes
-      * Clone the iTowns repo, e.g., `git clone git@github.com:iTowns/itowns2.git`
-      * Make changes in a branch, e.g., `git checkout -b my-feature`.
-
 ## Build the code
 
 * Download dependencies: `npm install`
 * Run the dev server:
-   * `npm start` (you can change the port: `npm start -- --port 3000` if you obtain an error message that is not related to the port being in use, please check that your npm version is >2.0 `npm --vesrion`)
+   * `npm start` (you can change the port: `npm start -- --port 3000`. if you obtain an error message that is not related to the port being in use, please update to last lts or stable version.
```
Comment (peppsac) [inline BUILDING.md]: Missing closing `)`

**Review by peppsac — APPROVED** [review]

**Review by gchoqueux — APPROVED** [review]

File: BUILDING.md (hunk 1 above)
Comment (autra) [reply BUILDING.md]: :-O correcting that now

**Comment by Mkonini** [comment]
Resize http://www.itowns-project.org/itowns2/examples/globe.html to be displayed in full screen

**Comment by gchoqueux** [comment]
For Globe example maybe do in other PR
The menu is broken too

**Comment by gchoqueux** [comment]
I do a new PR for example

**Comment by autra** [comment]
> Resize http://www.itowns-project.org/itowns2/examples/globe.html to be displayed in full screen

Not related to this PR.

**Pull request merged**